assets/css/login.css

```diff
@@ -0,0 +1,101 @@
+@import "/assets/css/nav.css";
+@import "/assets/css/reset.css";
+
+.container {
+  width: 100%;
+  height: calc(100vh - 8rem);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--back-grey);
+}
+
+.container-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--white);
+  padding: 3rem 5rem;
+  border-radius: 0.5rem;
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 1px 4px;
+}
+
```

Comment:
Could use variables for colours or mention some comment on the same if just used once.

Reply:
Ok, I'll update.